style.css

```diff
@@ -357,3 +300,57 @@ main .icon-md {
 .stories div:last-child {
   border-bottom: none;
 }
+
+/* responsiveness */
+
+.right-section {
+  display: none;
+}
+
+.tweet-btn,
+.spacer,
+.profile-shortcut,
+.nav-list,
+.nav-list a,
+.twitter {
+  display: none;
+}
+
+@media only screen and (min-width: 500px) {
+  .twitter,
+  .nav-list {
+    display: block;
+  }
+
+  .container {
+    display: grid;
+    grid-template-columns: auto 1fr;
+  }
+}
+
+@media only screen and (min-width: 700px) {
+  .twitter,
+  .nav-list,
+  .nav-list a {
+    display: block;
+  }
+
+  .container {
+    display: grid;
+    grid-template-columns: 0.3fr 1fr;
+  }
+}
```

Comment:
sometimes it's best to not use utility classes when trying to make an area responsive